### PR TITLE
feat: define CRD RAG under group evaluations 

### LINKED ISSUE #507: Define a new CRD RAG under group evaluations.arcadia.kubeagi.k8s.com.cn  provide lifecycle management of Evaluation Task

### PR DESCRIPTION
**Pull request by bjwswang**
#### Which issue(s) this PR fixes

Fixes #507 

#### Special notes for your reviewer

Changelogs:
1. move arcadia base CRD's controllers under controllers/base
2. add new group `evaluations`
3. add new CRD `RAG` under group `evaluations`  to evaluate `arcadia Application/Agent`

### CODE REVIEW

**Comment by bjwswang** [comment]
@Abirdcfly @0xff-dev any comments if I move controllers under `controllers/base/` ?

**Review by 0xff-dev — APPROVED** [review]

File: api/evaluation/v1alpha1/common.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2023 KubeAGI.
```
Comment (Abirdcfly) [inline api/evaluation/v1alpha1/common.go:2]: 2024

File: api/evaluation/v1alpha1/common.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	// Estimates context recall by estimating TP and FN using annotated answer and retrieved context.
+	ContextRecall MetricsKind = "context_recall"
+)
+
+type Metric struct {
+	// Kind of this Metric
+	Kind MetricsKind `json:"kind,omitempty"`
+
+	// Parameters in this Metrics
+	// See
```
Comment (Abirdcfly) [inline api/evaluation/v1alpha1/common.go:58]: see some url for detail?

**Review by Abirdcfly — APPROVED** [review]
I guess also need update apiserver/pkg/common/schema.go:122 init function, but apiserver do not use it.So we can add this to next pr.

File: api/evaluation/v1alpha1/common.go (hunk 2 above)
Comment (bjwswang) [reply api/evaluation/v1alpha1/common.go]: Parameters are quite different in different metric. We will show them in our document or guide user to ragas document. I have added some urls in metric definitions.

**Comment by bjwswang** [comment]
> I guess also need update apiserver/pkg/common/schema.go:122 init function, but apiserver do not use it.So we can add this to next pr.

Good thought. This is only a definition which may change.

**Pull request merged**